docs/07_migration-guides/01_v1-history-alternatives.md

```diff
@@ -25,7 +25,7 @@ Roborovski History API is implemented and ran by [Greymass Inc.](https://greymas
 
 ## What makes the Roborovski History API safe
 
-Roborovski History API has a high degree of safety because it is created by [Greymass Inc.](https://greymass.com/) which has been a credible and stable block producer and wallet developer (Anchor) company for EOS, WAX, TELOS, PROTON, FIO and other Antelope based chains.
+Roborovski History API has a high degree of safety because it is created by [Greymass Inc.](https://greymass.com/) which has been a credible and stable block producer and wallet developer (Anchor) company for EOS, WAX, TELOS, PROTON, FIO and other EOS based chains.
```

Comment:
Same here, or remove the reference altogether:
EOS based chain --> Antelope-based chain